packages/irmin/irmin.3.5.1/opam

```diff
@@ -0,0 +1,64 @@
+opam-version: "2.0"
+maintainer:   "thomas@gazagnaire.org"
+authors:      ["Thomas Gazagnaire" "Thomas Leonard"]
+license:      "ISC"
+homepage:     "https://github.com/mirage/irmin"
+bug-reports:  "https://github.com/mirage/irmin/issues"
+dev-repo:     "git+https://github.com/mirage/irmin.git"
+doc:          "https://mirage.github.io/irmin/"
+
+build: [
+ ["dune" "subst"] {dev}
+ ["dune" "build" "-p" name "-j" jobs]
+ ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml"   {>= "4.08.0"}
+  "dune"    {>= "2.9.0"}
+  "repr"    {>= "0.6.0"}
+  "fmt"     {>= "0.8.5"}
+  "uri"     {>= "1.3.12"}
+  "uutf"
+  "jsonm"   {>= "1.0.0"}
+  "lwt"     {>= "5.3.0"}
+  "digestif" {>= "0.9.0"}
+  "ocamlgraph"
+  "logs"    {>= "0.5.0"}
+  "bheap" {>= "2.0.0"}
+  "astring"
+  "mtime" {>= "1.0.0"}
```

Comment:
```suggestion
  "mtime" {>= "1.0.0" && < "1.5.0"}
```

```
### output ###
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlc.opt -w -40 -g -bin-annot -I src/irmin/.irmin.objs/byte -I /home/opam/.opam/4.14/lib/angstrom -I /home/opam/.opam/4.14/lib/astring -I /home/opam/.opam/4.14/lib/base64 -I /home/opam/.opam/4.14/lib/bheap -I /home/opam/.opam/4.14/lib/bigstringaf -I /home/opam/.opam/4.14/lib/digestif -I /home/opam/.opam/4.14/lib/either -I /home/opam/.opam/4.14/lib/eqaf -I /home/opam/.opam/4.14/lib/fmt -I /home/opam/.opam/4.14/lib/jsonm -I /home/opam/.opam/4.14/lib/logs -I /home/opam/.opam/4.14/lib/lwt -I /home/opam/.opam/4.14/lib/mtime -I /home/opam/.opam/4.14/lib/ocamlgraph -I /home/opam/.opam/4.14/lib/optint -I /home/opam/.opam/4.14/lib/ppx_irmin/internal-lib -I /home/opam/.opam/4.14/lib/repr -I /home/opam/.opam/4.14/lib/stdlib-shims -I /home/opam/.opam/4.14/lib/stringext -I /home/opam/.opam/4.14/lib/uchar -I /home/opam/.opam/4.14/lib/uri -I /home/opam/.opam/4.14/lib/uutf -I src/irmin/data/.irmin_data.objs/byte -intf-suffix .ml -no-alias-deps -open Irmin__ -o src/irmin/.irmin.objs/byte/irmin__Logging.cmo -c -impl src/irmin/logging.pp.ml)
# File "src/irmin/logging.ml", line 39, characters 15-31:
# 39 |       let dt = Mtime.Span.to_us (Clock.count start_time) in
#                     ^^^^^^^^^^^^^^^^
# Error: Unbound value Mtime.Span.to_us
```

See release notes: https://github.com/dbuenzli/mtime/blob/master/CHANGES.md

Reply:
If this is the right fix and it gets applied, don't forget to upstream the it.

Reply:
thanks @raphael-proust -- I actually fixed this with the 3.5.0 release and upstreamed but then forgot to include it when I branched for this patch release. :facepalm: